ml-agents/mlagents/trainers/policy/tf_policy.py

```diff
@@ -392,18 +400,50 @@ def get_update_vars(self):
         """
         return list(self.update_dict.keys())
 
-    def save_model(self, steps):
+    def checkpoint(self, model_reward: Optional[float] = None) -> None:
         """
-        Saves the model
-        :param steps: The number of steps the model was trained for
-        :return:
+        Checkpoints the model
```

Comment:
describe the argument model_reward

Reply:
Tracking the reward was based on @chriselion 's suggestion; my understanding of it is that it would be easier for a user to understand a past model based on its performance in training.

Reply:
^- I think the intention here is that we put the definition into the comment / docstring

Reply:
Yes, sorry I was not clear. I meant add a comment to the method.